go/libkb/services.go

```diff
@@ -140,6 +140,18 @@ func (t *BaseServiceType) GetAPIArgKey() string {
 
 func (t *BaseServiceType) IsDevelOnly() bool { return false }
 
+func (t *BaseServiceType) GetLogoKey() string {
+	t.Lock()
+	defer t.Unlock()
+	if t.displayConf == nil {
+		return ""
```

Comment:
It'd be nice if this returned the service key `t.Key()` as a fallback. But that might be annoying so fine without

Reply:
yeaaah I don't know how plumbing that would work, let's keep it as is for now and change it for a more reasonable fallback if something ever breaks